backend_new/src/controllers/map-layer.controller.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Controller,
+  Get,
+  Query,
+  UseGuards,
+  UsePipes,
+  ValidationPipe,
+} from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { MapLayersQueryParams } from '../dtos/map-layers/map-layers-query-params.dto';
+import { MapLayersService } from '../services/map-layers.service';
+import { MapLayerDto } from '../dtos/map-layers/map-layer.dto';
+import { defaultValidationPipeOptions } from '../utilities/default-validation-pipe-options';
+import { OptionalAuthGuard } from '../guards/optional.guard';
+import { PermissionGuard } from '../guards/permission.guard';
+
+@Controller('/mapLayers')
+@ApiTags('mapLayers')
+@UseGuards(OptionalAuthGuard, PermissionGuard)
```

Comment:
question: do we need to add mapLayers to the permission csv?

Reply:
Added!